src/index.js

```diff
@@ -327,9 +327,15 @@ export default class ImageTool {
    */
   uploadingFailed(errorText) {
     console.log('Image Tool: uploading failed because of', errorText);
-
+    
+    let errorMessage = 'Can not upload an image, try another.';
+    let errorJson = JSON.parse(errorText);
```

Comment:
You don't need to parse JSON, because it is already parsed in the method above. Use the `errorText` argument instead.